stdlib/asm/crypto/fri/frie2f4.masm

```diff
@@ -270,28 +338,152 @@ export.verify.4
     dup.2
     neq
 
+    # Save a word containing a fresh accumulator for Horner evaluating the remainder polynomial,
+    # a pointer to the evaluation point and a pointer to the location of the polynomial.
+    push.0.0 
+    exec.constants::tmp2 exec.constants::get_remainder_poly_address
+    exec.constants::tmp1 mem_storew
+    movup.4
+
     while.true
         # load [e0, e1, p, poe] from memory i.e. next query data (7 cycles)
-        push.0.0.0.0
         movup.4
         mem_loadw
         # => [poe, p, e1, e0, layer_ptr, rem_ptr, g, ...]
 
         # we now have everything to verify query p
-        exec.verify_query
+        exec.verify_query_64
 
         # prepare for next iteration (18 cycles)
         # => [x, x, x, x, x, x, x, x, x, x, g, ...]
```

Comment:
Based on the comments towards the end of `verify_query_64`, I expect the stack coming out of the procedure to have 12 garbage elements, but there are 10 garbage elements here

Reply:
Good catch, fixed it